data/iatemplates/mysql_down.yml

```diff
@@ -1,5 +1,9 @@
 ---
+<<<<<<< HEAD
+rules:
+=======
 templates:
+>>>>>>> origin/PMM-2.0
```

Comment:
Why did you commit unmerged files?

Reply:
Sometimes git does not list all changed files. I don't know why and when exactly. I'm fixing it now.

Reply:
It is immediately noticeable even if git does not work for some reason. Please review you own PRs too